src/js/components/FilterByTaskState.js

```diff
@@ -56,13 +57,22 @@ class FilterByTaskState extends React.Component {
   }
 
   render() {
+    let {inverseStyle} = this.props;
+
+    let dropdownMenuClassSet = classNames({
```

Comment:
yeah so i think `TaskView` should have the boolean and then everything else will have the defaultProps for className